README.md

```diff
@@ -202,6 +202,16 @@ def foobar(text):
 
 ## Changelog
 
+#### 1.0.0
+ - Feature: Default to not showing less usefull "full stacktrace"
+ - Feature: Add platform data to report
+ - Feature: Allow specification of a data_processor function that can alter the exception data
+ - Feature: Safer filenames (no colon characters)
+ - Bugfix: Handle all builtin exception types when attaching a message to an exception object
+ - Refactor: Combine repetitive code into create_exception_report
+ - Refactor: Simplify logging API
+ - Refactor: Split ExceptionReporter into component functions.
+
```

Comment:
drop `+[![build status](https://travis-ci.org/CircleUp/exception-reports.svg)](https://travis-ci.org/CircleUp/exception-reports)` at the top of the readme to get the cool build tag from travis

Reply:
will do